extensions/wikia/PhalanxII/hooks/PhalanxUserBlock.class.php

```diff
@@ -37,7 +37,7 @@ static public function blockCheck( User $user, $shouldLogBlockInStats = true, $g
 		$phalanxModel->setShouldLogInStats( $shouldLogBlockInStats );
 
 		$ret = $phalanxModel->match_user();
-		if ( $ret !== false ){
+		if ( $ret !== false ) {
 			if ( self::$checkEmail === true ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1066)